src/features/Apiexplorer/SubscribeRenderer/index.tsx

```diff
@@ -48,16 +54,16 @@ function SubscribeRenderer<T extends TSocketSubscribableEndpointNames>({
     }
 
     return request_data;
-  };
+  }, [reqData]);
 
   const handleClick = useCallback(() => {
-    unsubscribe();
-    subscribe(parseRequestJSON());
+    if (subscribe_ref.current) subscribe_ref.current.unsubscribe();
```

Comment:
```suggestion
    subscribe_ref?.current.unsubscribe();
```

Reply:
we never want the current reference to be undefined because if it is undefined and the user tries to clear or send a request as unsubscribe() function will throw an error  on undefined ref value. so if the ref.current doesn't have any value we want to skip to subscribe directly